playbooks/roles/deploy-osh/templates/horizon.yaml.j2

```diff
@@ -1,2 +1,27 @@
 ---
+{% if developer_mode %}
+conf:
+  apache2:
+    binary: apache2ctl
+    start_parameters: -DFOREGROUND -k start
+    site_dir: /etc/apache2/vhosts.d
+    conf_dir: /etc/apache2/conf.d
+    a2enmod:
+      - rewrite
+      - status
+      - version
+  horizon:
+    security: |
+      <Directory "/var/www">
+      Options Indexes FollowSymLinks
+      AllowOverride All
+      <IfModule !mod_access_compat.c>
+        Require all granted
+      </IfModule>
+      <IfModule mod_access_compat.c>
+        Order allow,deny
+        Allow from all
+      </IfModule>
+      </Directory>
+{%  endif %}
 {{ lookup('template','files/common-images-overrides.yml') | from_yaml | to_nice_yaml(indent=2) }}
```

Comment:
You can probably remove this common images overrides. It's useless now.

Reply:
out of scope for this patch? Separate patch better?

Reply:
The template could mask any of your overrides, and create duplicate keys. it's a bad bad idea.

Reply:
It's not related to this change, I would agree with separate PR (or at least separate commit)